tokio-compat/src/runtime/current_thread/runtime.rs

```diff
@@ -66,7 +69,7 @@ impl Handle {
     ///
     /// This function panics if the spawn fails. Failure occurs if the `CurrentThread`
     /// instance of the `Handle` does not exist anymore.
-    pub fn spawn_std<F>(&self, future: F) -> Result<(), tokio_executor::SpawnError>
+    pub fn spawn_std<F>(&self, future: F) -> Result<(), tokio_02::executor::SpawnError>
```

Comment:
would it make sense to combine these spawn errors? so we have a `tokio_compat::SpawnError`?

Reply:
I think these need to be the 0.1/0.2 types for API compatibility (although in the case of `spawn_std`, it's not drop-in compatible with 0.2 due to the name change...)

In practice, I suspect most code will use the `tokio::spawn` free fns on the appropriate version of `tokio`, so I am not sure if these are used that much. But, we should have them for API compatibility.